docs/en/command.md

```diff
@@ -0,0 +1,541 @@
+# Working with database
+
+## Creating a table
+
+To create a table, you can use the `Yiisoft\Db\Command\CommandInterface::createTable()` method.
+
+The following example shows how to create a table.
+
+```php
+<?php
+
+declare(strict_types=1);
+
+use Yiisoft\Db\Connection\ConnectionInterface;
+
+/** @var ConnectionInterface $db */
+$db->createCommand()->createTable('{{%customer}}', [
```

Comment:
Need add string - better use db-migrate package, but you can create database with using columnSchemaBuilder

Reply:
And need add example with string definitions for column